src/server/replica.cc

```diff
@@ -129,11 +140,49 @@ vector<vector<unsigned>> Partition(unsigned num_flows) {
 std::string Replica::MasterContext::Description() const {
   return absl::StrCat(host, ":", port);
 }
+static SSL_CTX* CreateSslCntx() {
```

Comment:
why / how is this function different than CreateSslCntx in dragonfly_listerner.cc?

Reply:
Mostly it's creating a client context which is different than a server context, I'd change both functions' names to reflect that.